[node] JSON file cannot be resolved without explicit ".json" extension when using require

#### Environment details

  - OS: Windows 10
  - Node.js version: 9.11.1
  - npm version: 6.1.0
  - `@google-cloud/datastore` version: 2.0.0

#### Steps to reproduce

  1. compile code with webpack


When compiling the code, the next error occurs:
```
ERROR in ./node_modules/@google-cloud/datastore/src/v1/datastore_client.js
Module not found: Error: Can't resolve './datastore_client_config' in '...\node_modules\@google-cloud\datastore\src\v1'
```

Due to require not having explicit '.json' extension, compiler fails to find the file
```javascript
// datastore_client.js
const gapicConfig = require('./datastore_client_config');
```

![image](https://user-images.githubusercontent.com/2654812/49335717-85a87f00-f5b8-11e8-9f63-49c1223d8406.png)

--- a/src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSImportSectionTransformer.java
+++ b/src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSImportSectionTransformer.java
@@ -58,7 +58,7 @@ public class NodeJSImportSectionTransformer implements ImportSectionTransformer 
     InterfaceModel apiInterface = context.getInterfaceModel();
     InterfaceConfig interfaceConfig = context.getInterfaceConfig();
     String configModule = context.getNamer().getClientConfigPath(interfaceConfig);
-    imports.add(createImport("gapicConfig", "./" + configModule));
+    imports.add(createImport("gapicConfig", "./" + configModule + ".json"));
     imports.add(createImport("gax", "google-gax"));
     imports.add(createImport("merge", "lodash.merge"));
     imports.add(createImport("path", "path"));

--- a/src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline
+++ b/src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline
@@ -3214,7 +3214,7 @@ module.exports.LibraryServiceClient = LibraryServiceClient;
 
 'use strict';
 
-const gapicConfig = require('./library_service_client_config');
+const gapicConfig = require('./library_service_client_config.json');
 const gax = require('google-gax');
 const merge = require('lodash.merge');
 const path = require('path');

--- a/src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_multiple_services.baseline
+++ b/src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_multiple_services.baseline
@@ -185,7 +185,7 @@ module.exports.default = Object.assign({}, module.exports);
 
 'use strict';
 
-const gapicConfig = require('./decrementer_service_client_config');
+const gapicConfig = require('./decrementer_service_client_config.json');
 const gax = require('google-gax');
 const merge = require('lodash.merge');
 const path = require('path');
@@ -503,7 +503,7 @@ const Empty = {
 
 'use strict';
 
-const gapicConfig = require('./incrementer_service_client_config');
+const gapicConfig = require('./incrementer_service_client_config.json');
 const gax = require('google-gax');
 const merge = require('lodash.merge');
 const path = require('path');

--- a/src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_no_path_templates.baseline
+++ b/src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_no_path_templates.baseline
@@ -176,7 +176,7 @@ module.exports.NoTemplatesApiServiceClient = NoTemplatesApiServiceClient;
 
 'use strict';
 
-const gapicConfig = require('./no_templates_api_service_client_config');
+const gapicConfig = require('./no_templates_api_service_client_config.json');
 const gax = require('google-gax');
 const merge = require('lodash.merge');
 const path = require('path');

